src/plugins/aws/ssh.ts

```diff
@@ -22,11 +22,10 @@ import {
   AwsSshRoleRequest,
 } from "./types";
 
-/** Maximum number of attempts to start an SSH session
- *
- * Each attempt consumes ~ 1 s.
+/**
+ * It can take up to 1 minute for access to propagate on AWS, so set the time limit to 10 minutes.
```

Comment:
What's the motivation to change from 30 seconds to 10 minutes?

I'd be surprised if any end user is willing to wait 10 minutes.

Reply:
The timeout is based on existing comments in ssh/index.ts:

AWS takes about 8 minutes, GCP takes under 1 minute to fully resolve access after it is granted.
During this time, calls to `aws ssm start-session` / `gcloud compute start-iap-tunnel`
will fail randomly with an various error messages.

See also this PR, which also mentions 10 minutes:
https://github.com/p0-security/p0cli/pull/19

Reply:
@gergas3 @nbrahms I can change the timeout if you want. Just let me know what to set it to.

Reply:
Was it a typo in https://github.com/p0-security/p0cli/pull/19 then? I didn't observe such a long propagation time for AWS. 30 seconds makes more sense.

GCP takes significantly longer, 1 minute seems about right but it's my guess would be it's not >99th percentile. We continue with 1m imo.

Reply:
Changed it to 30 seconds.

Reply:
The full info is that it takes AWS 30 minutes to get to 100% success rate. But we don't need to wait this long. We only need to wait until the chance that a single SSH attempt succeeds is high.

I'm not sure what we've changed in terms of delays, as those will affect this. But, when we were hammering it repeatedly in a loop, that took about 12 seconds.